packages/yoga/src/Heading/web/Heading.jsx

```diff
@@ -43,11 +43,13 @@ Heading.propTypes = {
     PropTypes.node,
   ]),
   noPadding: bool,
+  bg: PropTypes.string,
```

Comment:
A more meaningful name:
```suggestion
  backgroundColor: PropTypes.string,
```
Also, maybe we can start using [transient props](https://styled-components.com/docs/api#transient-props) naming so these "style" props don't go to the DOM.

```suggestion
  $backgroundColor: PropTypes.string,
```

Reply:
This name is part of the styled system, we can also use the `backgroundColor` and `bgColor` attributes to define the background color although they are not listed in the prop-types. When we start using typescript it will be possible to list all these props automatically.
As these props are part of the styled system, they are not being injected into the DOM.